beacon-chain/blockchain/process_block.go

```diff
@@ -696,6 +651,12 @@ func (s *Service) lateBlockTasks(ctx context.Context) {
 		log.WithError(err).Debug("could not update next slot state cache")
 	}
 
+	go func() {
+		if err := s.updateShufflingCaches(ctx); err != nil {
```

Comment:
Why do we do this both here and in `onBlock`? `insertBlockToForkchoiceStore` is called inside `onBlock`, meaning that shuffling will be called twice.

Reply:
this is not part of `onBlock`

Reply:
oops, I misread it on Github, sorry